### ISSUE TITLE
Incorrect response in Choice Chips of a custom function

### ISSUE THREAD

**Issue by ALazutin10** [FlutterFlow/flutterflow-issues#1184]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

The choice chips are returning a null or empty result which is defined as "No Slots Available" 

https://github.com/FlutterFlow/flutterflow-issues/assets/119873933/777c10ee-7b15-4993-a2f7-8b58afbb2975

The screen recording shows the result 

### Expected Behavior

It is expected that instead of "No Slots Available" there would be availability slots that are available and returned correctly. 
The custom function that is used to achieve this returns the result correctly, but this won't show in the choice chips.

Link to custom function: https://app.flutterflow.io/project/beauty-ebbo16?tab=customCode&customCodeTab=functions

### Steps to Reproduce

Create a project, setup firebase and use the same database schema as is set up in the project link above. 
Use the custom function in the link above and set up the screen accordingly.
Test the linked custom function, see how it delivers results correctly. 
Then run the app in test mode and see how it doesn't correctly show any available slots. 

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

ITFXlvHlz85IvuZE7qrycMF4iWIwGVgLaJJEktYbdzwYCLbpPe9/fsn4QBNCYNyrY3lhHGGJgmYH0taNk+ztLPAqF0ycG75HyJN2Sxbjc3q8RbKSDYeWXEZSGp9YIFSX1Kavgx0lBNdpdE0t112YIeq9Uh7fGZu/Zwh9f6PHaOI=

### Context

I need to show the correct availabilities to build a fully functional and working application. This issue is preventing me from doing this. I would normally diagnose and fix this myself, but since all functions are working correctly, this issue must lie elsewhere

### Visual documentation

Added above
<img width="875" alt="Screenshot 2023-07-27 at 19 27 44" src="https://github.com/FlutterFlow/flutterflow-issues/assets/119873933/0b9e17a7-2c1c-409e-a6d5-c195bc99253a">
<img width="858" alt="Screenshot 2023-07-26 at 19 20 57" src="https://github.com/FlutterFlow/flutterflow-issues/assets/119873933/d8dadc04-4960-4ae1-a2e4-acf21649f724">

### Environment

```markdown
- FlutterFlow version: 3.1
- Platform:macOS desktop
- Browser name and version: Chrome 114.0.5735.198 (Official Build) (x86_64)
- Operating system and version affected: All
```

**Comment by ALazutin10** [comment]
Can a team member address this issue?

**Comment by aemelyanovff** [comment]
@ALazutin10 This issue tracker is intended for general bugs that are applicable to all users. Because this error appears to be specific to your project and the steps are not reproducible, this is classified as a support request and will be closed. Please use chat support or contact support [at] flutterflow.io.

**Comment by ALazutin10** [comment]
Anton, thanks for your message. 
I was directed to you by David from support who said that I should post my issue there, instead of support as he is not capable of solving anything. They were checking then just dropped it. Who can I reach out to in order to solve this issue? 
I would really appreciate it if you could direct me to the right person as it’s been a nightmare with support and trying to reach out to no avail. 

Regards
Alexander Lazutin
Director | ICarta Technologies



On 9 Aug 2023, at 20:33, Anton Emelyanov ***@***.***> wrote:


@ALazutin10 This issue tracker is intended for general bugs that are applicable to all users. Because this error appears to be specific to your project and the steps are not reproducible, this is classified as a support request and will be closed. Please use chat support or contact support [at] flutterflow.io.

**Comment by davidflutterflow** [comment]
@ALazutin10 ,

Thank you for reaching out and sharing your concerns. We understand that your recent interaction with our support team may not have met your expectations. Allow me to clarify the situation:

Our conversation was concluded due to a shift in topics, addressing multiple unrelated issues simultaneously, extending the conversation with lengthy messages, and raising grievances beyond the scope of our support's capabilities.

For a smoother and more efficient support experience, we kindly ask you to adhere to our support guidelines, which can be reviewed in our [Customer Support Policy](https://docs.flutterflow.io/miscellaneous/customer-support-policy).

If your concern falls within our support's purview, please don't hesitate to reach out with one specific issue at a time, and we'll be glad to assist you.

Regarding the matter you insist is a bug, it's essential to understand that we were unable to reproduce the issue on our end. As such, we cannot classify it as a bug. If you remain convinced that it is a bug, we recommend reproducing the issue and adding it to our issue tracker for further evaluation.

In conclusion, our support team is here to help you within the framework of our capabilities. While we may not be able to fix an alleged bug instantly, we're committed to assisting you with all inquiries that align with our support policy.

We value your collaboration in helping us help you. If you have any questions or need further assistance, please don't hesitate to reach out.

Best regards,
David

**Comment by ALazutin10** [comment]
David, you closed this issue because you didn’t bother to look into it. I was really disappointed and annoyed with your attitude. In the chat flow, an agent by the name of Marshall was looking into the issue, asking me for more evidence and testing to see what was wrong. You, however, just closed it without allowing me to reply - this is woeful customer service!

Regarding the issue itself, Anton has directed me back to support and you come back to say you cannot help. Why? 
It is clearly an issue, I don’t know why you’re not seeing it. If you cannot understand, schedule a zoom call and I will guide you. I’m keen to meet you halfway and provide information and I would like to see the same from you, not just cutting off conversations and not replying! 

To reproduce it in a new project, you will have to make all the functionality the same, it will take a while. A much easier approach will be to test my project and see for yourself. Since you have access to the app builder, you can test the custom function by the name of ShowAvailabilities and see that it works correctly. Then, test the app in test mode and via the vendor page, try to pick a day and time. It shows nothing is available. Test using the service by the name of “Haircut” since availability there is set to everyday and many slots should come, but they don’t. -> And now we can clearly see an issue

Supporting screenshots and video recording were provided. Take some time and effort to look into it and come back to me. Don’t send more excuses, that is embarrassing and looks awful for your brand. 

Regards
Alexander Lazutin
Director | ICarta Technologies



Build your technology: www.icartatechnologies.com <http://www.icartatechnologies.com> 





On 11 Aug 2023, at 13:29, David ***@***.*** ***@***.***> > wrote:


@ALazutin10 ,

Thank you for reaching out and sharing your concerns. We understand that your recent interaction with our support team may not have met your expectations. Allow me to clarify the situation:

Our conversation was concluded due to a shift in topics, addressing multiple unrelated issues simultaneously, extending the conversation with lengthy messages, and raising grievances beyond the scope of our support's capabilities.

For a smoother and more efficient support experience, we kindly ask you to adhere to our support guidelines, which can be reviewed in our Customer Support Policy <https://docs.flutterflow.io/miscellaneous/customer-support-policy> .

If your concern falls within our support's purview, please don't hesitate to reach out with one specific issue at a time, and we'll be glad to assist you.

Regarding the matter you insist is a bug, it's essential to understand that we were unable to reproduce the issue on our end. As such, we cannot classify it as a bug. If you remain convinced that it is a bug, we recommend reproducing the issue and adding it to our issue tracker for further evaluation.

In conclusion, our support team is here to help you within the framework of our capabilities. While we may not be able to fix an alleged bug instantly, we're committed to assisting you with all inquiries that align with our support policy.

We value your collaboration in helping us help you. If you have any questions or need further assistance, please don't hesitate to reach out.

Best regards,
David
